server/src/main/java/com/google/coffeehouse/StorageHandler.java

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.coffeehouse;
+
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.Statement;
+
+public class StorageHandler {
+
+  static void getPersonQuery(DatabaseClient dbClient, String userId) {
+    Statement statement = 
+        Statement.newBuilder(
+                "SELECT userId, email, nickname, pronouns "
+                  + "FROM Persons "
+                  + "WHERE userId = @userId")
+            .bind("userId")
+            .to(userId)
+            .build();
+    try (ResultSet resultSet = dbClient.singleUse().executeQuery(statement)) {
+      while (resultSet.next()) {
+        System.out.printf(
+          "User ID: %s || Email: %s || Nickname: %s || ",
+          resultSet.getString("userId"),
+          resultSet.getString("email"),
+          resultSet.getString("nickname"));
```

Comment:
While printing is very important for testing, should we be printing it to the console after we know it's querying correctly after it's been pushed to coffea? (maybe we do, I just wanted to point it out for security reasons)

Reply:
^^^ Yes, if we need it for now that's fine but eventually we should try to avoid any PII (Personally Identifiable Information) going in the system logs

Reply:
I removed the print statements and instead return a string instead (which will be refactored once I push this code and update it to reflect any of Joey's changes).